### ISSUE TITLE
QUESTION: How do I keep subtitles at 24fps?

### ISSUE THREAD

**Issue by fugueink** [CCExtractor/ccextractor#1305]
CCExtractor version: 0.88

# In raising this issue, I DO NOT confirm the following:

- [ ] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [ ] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [ ] I have checked that the issue I'm posting isn't already reported.
- [ ] I have checked that the issue I'm porting isn't already solved and no duplicates exist in [closed issues](https://github.com/CCExtractor/ccextractor/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/CCExtractor/ccextractor/issues)
- [ ] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.
- [ ] I have used the latest available version of CCExtractor to verify this issue exists.
- [ ] I have ticked all the boxes in this section and to prove it I'm deleting the section completely to remove boilerplate text.

# Necessary information

- Is this a regression (i.e. did it work before)? {YES/NO}
- What platform did you use? {Window/Linux/Mac}
- What were the used arguments? `{replace with the arguments}`

# Video links

* {Replace with a link to a video file}

# Additional information

I apologize in advance, but I have tried for more than a decade (and I think more than two) to understand GitHub and the way it works. I can't parse it. I wish OpenSource software used something else. Generally I just live with whatever problem, but my sister and I *NEED* subtitles.

I am using CCExtractor 0.88 via the Terminal in macOS 10.13.6 (High Sierra). It's the highest my software allows. It always changed my 24 fps subtitles into 30 fps, which requires me adjusting each blasted SRT by hand in Jubler. I don't have that kind of time.

I looked for a switch in CCExtractor to make it stay with 24 fps (actually 23.9-something), but to no avail.

Can anyone help?

**Comment by cfsmp3** [comment]
CCExtractor doesn't do FPS - it just uses the the timing from the input video.
But if you think there's a bug somewhere with a specific video please share a link to it. We can't fix anything we can't reproduce.

**Comment by fugueink** [comment]
Well, I found a way to fix it quickly in Jubler, so I guess there's no need to tweak CCExtractor.

I am confused by your saying that it uses the input video for the timing. That's how I figured out what the problem was: CCExtractor said
`
Total frames time:	  00:34:47:085  (62550 frames at 29.97fps)`

I knew, however, that the file was 24fps.

CCE is also always saying  things like:
`Changed fps using NAL to: 23.976024`

Sounds like it's doing something to the fps. . . .

Thanks anyway! My apologies for never being able to figure out this help system.
